uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/components/my-leave.js

```diff
@@ -19,7 +19,9 @@ define([
         toDate: new Date(),
         showDatePickerFrom: false,
         showDatePickerTo: false,
-        isChangeExpanded: false
+        isChangeExpanded: false,
+        selectedResponse: "1",
+        isAdmin: true
```

Comment:
This one must be `false` when this is going to be merged, otherwise you're going to interfere with @dhavalsharma 's work